eslint.config.mjs

```diff
@@ -0,0 +1,64 @@
+import { fixupConfigRules } from "@eslint/compat";
+import reactRefresh from "eslint-plugin-react-refresh";
+import globals from "globals";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+const GLOBALS_BROWSER_FIX = Object.assign({}, globals.browser, {
+  AudioWorkletGlobalScope: globals.browser["AudioWorkletGlobalScope "],
+});
+delete GLOBALS_BROWSER_FIX["AudioWorkletGlobalScope "];
```

Comment:
What is this for?

Reply:
there was an issue and warnings when i tried to upgrade eslint v9

and found this link https://github.com/sindresorhus/globals/issues/239